### PR TITLE
Mdkessler patch 11

### PR DESCRIPTION
**Pull request by mdkessler**
**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [x] Yes, I added myself or am already a contributor

**Any more details?**

### CODE REVIEW

File: content/10.blackbox.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
 ## Tip 8: Your deep learning models can be more transparent {#blackbox}
 
-Model interpretability is a broad concept.
+While model interpretability is a broad concept, in much of the machine learning literature (including in our guidelines), the concept of model interpretability refers to the ability to identify the discriminative features that influence or sway the predictions.
```
Comment (Benjamin-Lee) [inline content/10.blackbox.md:3]: Just to help keep word count down:

```suggestion
While model interpretability is a broad concept, in much of the machine learning literature (including in our guidelines), it refers to the ability to identify the discriminative features that influence or sway the predictions.
```

**Review by Benjamin-Lee — APPROVED** [review]

File: content/10.blackbox.md [hunk 2]
```diff
@@ -1,25 +1,25 @@
 ## Tip 8: Your deep learning models can be more transparent {#blackbox}
```
Comment (cgreene) [inline content/10.blackbox.md:1]:
```suggestion
## Tip 8: Deep learning models can be made more transparent {#blackbox}
```

**Review by cgreene — APPROVED** [review]
One small suggestion re: second person.

**Comment by Benjamin-Lee** [comment]
Thanks for the suggestion @cgreene. Not sure if we're going to have to make all the tip titles third person but we'll cross that bridge when we come to third person normalization

**Pull request merged**